specification/eventhub/resource-manager/Microsoft.EventHub/preview/2024-05-01-preview/examples/EventHubs/EHEventHubCreate.json

```diff
@@ -31,8 +30,11 @@
           "sizeLimitInBytes": 10485763
         },
         "retentionDescription": {
-          "cleanupPolicy": "Delete,Compact",
+          "cleanupPolicy": "DeleteOrCompact",
```

Comment:
Let's keep the default policy for the basic example and add new example for DeleteOrCompact policy.
EHEventHubCreate.json -- default retention & message timestamp policy
EHEventHubWithCompactPolicyCreate.json
EHEventHubWithDeletePolicyCreate.json
EHEventHubWithDeleteOrCompactPolicyCreate.json -- new example.

Reply:
Done